### ISSUE TITLE
Add page search

### ISSUE THREAD

**Issue by EmilySeville7cfg** [526avijitgupta/gokarna#93]
It would be nice to have an ability to search posts and pages. It is useful for sites with a lot of content.

**Comment by yashmehrotra** [comment]
Yes, thanks for the suggestion. Its definitely in our roadmap. Although I can't commit a time to you for this we'll hopefully add this feature in the next few releases.

Adding client-side search is quite complex as we would have to think about the UI, research about doing it without vastly increasing the overall size of the served content & also try to fit most of the search use-cases.

We are also open to ideas for this implementation. For the time being I suggest to use any search engine to do this (`site:example.com search query`, yes I know its not ideal 😞 )

**Comment by mansoorbarri** [comment]
Just wanted an update on this because my website has many articles and i'm thinking of changing the theme because of lack of search
